python/src/opendp/polars.py

```diff
@@ -1,6 +1,163 @@
+from __future__ import annotations
+from typing import Literal, Tuple
+from opendp._lib import import_optional_dependency, lib_path
 from opendp.mod import assert_features
 
 
+pl = import_optional_dependency("polars", raise_error=False)
+
+
+if pl is None:
+    def _register_expr_namespace(_name):
+        return lambda c: c
+else:
+    _register_expr_namespace = pl.api.register_expr_namespace
+
+
+@_register_expr_namespace("dp")
```

Comment:
Will this even be loaded if Polars is not in use? I wonder if this could be simplified.
```suggestion
pl = import_optional_dependency("polars")

@pl.api.register_expr_namespace("dp")
```

Reply:
I think `pl.api` would fail as `pl` is None when Polars is not installed.